src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const repaired = {};
```

Comment:
You can be more specific.
```suggestion
  const repairedRobot= {};
```